README.md

```diff
@@ -1,14 +1,27 @@
-# voyage-tasks
+# Chingu Voyage 40 Toucans Team 10
 
-Your project's `readme` is as important to success as your code. For 
-this reason you should put as much care into its creation and maintenance
-as you would any other component of the application.
+## Instructions:
 
-If you are unsure of what should go into the `readme` let this article,
-written by an experienced Chingu, be your starting point - 
-[Keys to a well written README](https://tinyurl.com/yk3wubft).
+**To install the dependencies**:
 
-And before we go there's "one more thing"! Once you decide what to include
-in your `readme` feel free to replace the text we've provided here.
+```cmd
+npm install
+```
 
-> Own it & Make it your Own!
+**To check the style if it follows the style guide**:
+
+```cmd
+npm run lint
+```
+
+**To start it in development**:
+
+```cmd
+npm run dev
```

Comment:
Do I enter this command before I start the development?

Reply:
I meant to start the program in development will change!